app.py

```diff
@@ -46,11 +48,26 @@ def jsonify(self, requestIpAddress):
             "id": self.id}
     def csvify(self):
         return f"{self.name};{self.ipAddress};{self.startTime.strftime(dateTimeFormat)};{self.endTime.strftime(dateTimeFormat)};{self.id}\n"
+    def getMinutesReserved(self):
+        beginTime = self.startTime if datetime.now() < self.startTime else datetime.now()
+        timeDelta = self.endTime - beginTime
+        return timeDelta.days * 24 * 60 * 60 + timeDelta.seconds // 60
```

Comment:
I think its more user friendly to give them the extra minute, since the user cannot see the seconds left or reserve those seconds. If they have 59 minutes and 30 seconds reserved, they only see 59, so they must be able to reserve an extra minute